packages/clerk-js/src/utils/captcha.ts

```diff
@@ -76,9 +74,12 @@ export async function loadCaptcha(url: string) {
   if (!window.turnstile) {
     try {
       await loadScript(url, { defer: true });
-    } catch (_) {
+    } catch {
       // Rethrow with specific message
-      clerkFailedToLoadThirdPartyScript('Cloudflare Turnstile');
+      console.error('Clerk: Failed to load the CAPTCHA script from the URL: ', url);
+      throw {
+        captchaError: 'captcha_script_failed_to_load',
+      };
```

Comment:
🙃 I would suggest we throw a `ClerkError` (an existing one or create a new one) instead of throwing an object. In order to provide a better error handling in the future i think that we will need to use our Error classes.

Reply:
It's a bit complicated! We don't want everything to throw here. We actually send the `captchaError` error value to the FAPI.